GoMeet/src/main/java/com/google/sps/servlets/LocationServlet.java

```diff
@@ -60,7 +60,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       ServletUtil.sendErrorResponse(
           response, HttpServletResponse.SC_BAD_REQUEST, ErrorMessages.BAD_REQUEST_ERROR_LOCATION);
     } catch (MaxEntitiesReachedException e) {
-      // TODO: Handle max entities reached.
+      ServletUtil.sendErrorResponse(
+          response, HttpServletResponse.SC_BAD_REQUEST, ErrorMessages.MAX_ENTITIES);
```

Comment:
I would also like some feedback on whether you think this is the right way to signal that the max number of entities has been reached, particularly, whether I've used an appropriate error code.

Reply:
Seems appropriate to me